nbook.nim

```diff
@@ -13,4 +13,5 @@ var book = newBookFromToc("nimibook", "book"):
     draft("Draft chapter", "draft.md")
 
 book.git_repository_url = "https://github.com/pietroppeter/nimibook"
-nbBookTasks
\ No newline at end of file
+book.keep = @["tocexample"]
```

Comment:
do we need this, since it appears in the toc?

Reply:
No, we don't it was just to check that the ``book.keep`` mechanism was working